### ISSUE TITLE
Linux executable

### ISSUE THREAD

**Issue by oskarkk** [midnightfreddie/nbt2json#22]
It would be nice if there was a Linux executable available for download.

**Comment by MimzyFang** [comment]
yeah I would need that for updating my server to 1.20.1 and resetting the end and nether with better end and better nether enabled which requires modifying the level.dat file
